app/jobs/reservation_cancel_job.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class ReservationCancelJob < ApplicationJob
+  queue_as :default
+
+  def perform(*_args)
```

Comment:
Probably you don't want to pass any arguments here.
```suggestion
  def perform
```